### PR TITLE
feat(api): Create query param to show disabled ticket integrations

### PR DESCRIPTION
**Pull request by schew2381**
We want to show disabled integrations on the alert rule action dropdown. Each disabled integration would have a button on the right linking to respective integration setup page.

This change allows us to pass a query parameter that will then include all disabled ticket actions in the response.

### Mockup
Untitled

### CODE REVIEW

File: src/sentry/api/endpoints/project_rules_configuration.py [hunk 1]
```diff
@@ -81,4 +83,9 @@ def get(self, request: Request, project) -> Response:
 
         context = {"actions": action_list, "conditions": condition_list, "filters": filter_list}
 
+        if request.GET.get("includeAllTickets") is not None:
```
Comment (schew2381) [inline src/sentry/api/endpoints/project_rules_configuration.py:86]: Very open to alternative names for the query parameter `includeAllTickets` or the response key `disabledTicketActions`

Comment (scttcper) [reply src/sentry/api/endpoints/project_rules_configuration.py:86]: does it need to take into account `can_create_tickets`

Comment (schew2381) [reply src/sentry/api/endpoints/project_rules_configuration.py]: ah good catch, added that in

**Review by ceorourke — APPROVED** [review]
Change seems fine, will the disabled integrations in the list be greyed out / obvious that they're disabled? You might consider linking to the integration details page rather than the setup docs, since to re-enable you just need to re-install.

**Comment by schew2381** [comment]
> Change seems fine, will the disabled integrations in the list be greyed out / obvious that they're disabled? You might consider linking to the integration details page rather than the setup docs, since to re-enable you just need to re-install.

Ah yes that'd be better, I'll do that instead

**Pull request merged**